pkg/stream/receiver.go

```diff
@@ -43,7 +43,7 @@ type msgReceiver struct {
 }
 
 // Read read fill message with received data and return the number of read bytes or error. If no message is present it returns 0 and io.EOF. If the message is complete it returns 0 and nil, in that case successive calls to Read will returns a new message.
```

Comment:
It might be expected to return the number of read bytes, even if the buffer is successfully read

Reply:
When multiple messages are readed by receiver `read` method returns  every times the number of readed bytes. If the message is fully readed the next call to `read` will return 0 or io.EOF in case of no more messages are present on the wire.

Reply:
Probably the comment need to be improved